### ISSUE TITLE
Add a tutorial step by step

### ISSUE THREAD

**Issue by macagua** [cornernote/yii2-dashboard#6]
Hi @cornernote 

First of all thanks for the extension yii2-dashboard, seems very potential for a development I'm doing with Yii2 framework.

I have read the yii2-dashboard extension documentation, I used the yii2-dashboard demonstration website before installing it locally on my Yii2 basic project template.

I have managed [to enable settings yii2-dashboard](https://cornernote.github.io/yii2-dashboard/docs/module-configuration/) and via [yii2 migrations](http://www.yiiframework.com/doc-2.0/yii-db-migration.html) recreate the structure of your website demonstration following screenshots that have published.

Result of these practices have published this Yii2 project here: https://github.com/macagua/my_yii2dashboard_demo

Yii2 project have the following characteristics:
- Yii2 basic template.
- The extension _yii2-dashboard_ enabled via [yii2 module configuration](https://github.com/macagua/my_yii2dashboard_demo/blob/master/config/web.php#L51).
- Dashboard structures enabled via [yii2 migrations (INSERT SQL)](https://github.com/macagua/my_yii2dashboard_demo/tree/master/migrations).
- Enabled to Layout and Panel called **Example** following the _yii2-dashboard_ demonstration/documentation website.

Until now it works perfectly the extension of _yii2-dashboard_,  And in **my_yii2dashboard_demo** repository currently me in this screenshot:

![Dashboard Update](https://cloud.githubusercontent.com/assets/51875/8636682/4873cbe2-28ae-11e5-9feb-0a41cd14efc1.png)

Then in the extension documentation it does not indicate how to achieve this view:
![Dashboard View](https://cloud.githubusercontent.com/assets/51875/8636670/21febed6-28ae-11e5-8fc7-dc57e5bbc422.png)

As my object play configurations and source code of the demonstration site, to understand the operation of this extension:

As I understand the demonstration site you are consulted the workflow of a "Job" that contains three status / transitions "requested", "scheduled" and "collected".

So here I have my questions for you:
- You use the extension yii2-workflow?
- As I generate the representation "Dashboard View" with the panel "Collection Scheduled"? I explain better, **'jobStatus' => 'JobWorkflow / scheduled'** and **'JobType' => 'collect'** are Controller actions? I guess require some kind of view?

Please, at this point it would be great to help me to understand the operation of the extension, as I do not get some yii2 source code in reference to achieve the proper functioning of the panels of the dashboard extension.

Besides that I can help you make an initial version of a tutorial for yii2-dashboard.

Thanks

**Comment by cornernote** [comment]
Hi @macagua,

Thanks for your feedback.  I will try to answer your questions below, but please let me know if I miss anything.

> Then in the extension documentation it does not indicate how to achieve this view:

You need to write your own panel view file, as described here (see Panel View):
https://cornernote.github.io/yii2-dashboard/docs/panels/

> You use the extension yii2-workflow?

Yes, but it's not required.  It just happened to be used for the example.

> As I generate the representation "Dashboard View" with the panel "Collection Scheduled"? I explain better, 'jobStatus' => 'JobWorkflow / scheduled' and 'JobType' => 'collect' are Controller actions? I guess require some kind of view?

Yes, but these are just for the screenshot/demo, you don't need these in your application (if you do, it's not usage of dashboard, it's usage of workflow).

**Comment by macagua** [comment]
Hi @cornernote

For the panel "Collection Scheduled" example do you using the GridView::widget for create the grid table?

**Comment by cornernote** [comment]
@macagua, yep I used `GridView::widget`.  You can however render anything you want in the panel's view.
